utils/rpms/daos.spec

```diff
@@ -228,11 +235,14 @@ This is the package needed to build software with the DAOS library.
       --install-sandbox=%{buildroot} \
       %{buildroot}%{_prefix}         \
       %{buildroot}%{conf_dir}        \
-      USE_INSTALLED=all               \
```

Comment:
Ditto.

Reply:
It should still be set when cov is turned off.

Reply:
Per above.

Reply:
will undo